README.md

```diff
@@ -1,230 +1,234 @@
-# electricitymap [![Slack Status](http://slack.tmrow.co/badge.svg)](http://slack.tmrow.co)
```

Comment:
Hmm what happened here? How come the whole file was modified?
Can you try to fix that, or maybe revert and only modify the relevant parts?
Thanks!